auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/model/Entitlement.java

```diff
@@ -36,6 +36,7 @@
     CDP_LIST_FILTERING,
     CDP_EMBEDDED_DATABASE_ON_ATTACHED_DISK,
     CDP_DATA_LAKE_LOAD_BALANCER,
+    CDP_DATA_HUB_LOAD_BALANCER,
```

Comment:
Here's a question: How many entitlements do we want to have? I created the data lake specific LB entitlement because there are two uses cases for the LB: data lake HA and semi-private networks. And I wanted to be able to enable one or the other. But for the data hub I don't think we currently have a second use case for it. We'll only want to enable it if we're using semi-private networks. So should we have a data hub specific entitlement to be consistent with the data lake specific entitlement? Or would it be sufficient to just check for the semi-private network specific entitlement, which I added as part of https://github.com/hortonworks/cloudbreak/pull/9675? I could see the argument being made either way. Just wanted to bring it up.

Reply:
Thanks for bringing it up. Now we have 3 entitlements it seems. I had started this patch before looking at #9675. The reason DH is different than DL is just in case we have a bug for DHs but not for DLs. This is unlikely, but I wanted to be cautious. Let me rebase this on top of #9675, and get rid of this entitlement in favor of CDP_PUBLIC_ENDPOINT_ACCESS_GATEWAY

Reply:
Actually ended up rebasing on top of #9675 and getting rid of this entitlement, because having 3 is a bit too much. 
Did you add CDP_PUBLIC_ENDPOINT_ACCESS_GATEWAY to the thunderhead code base yet?